debian/Dockerfile

```diff
@@ -23,6 +22,7 @@ RUN buildDeps=' \
 		autoconf \
 		automake \
 		bzip2 \
+		curl \
```

Comment:
We've still got `wget` listed in the installed deps above -- if we switch over to using that instead of `curl` (`wget '...' -O '...'` instead of `curl -fsSL '...' -o '...'`) then we can skip this too. :+1: